lib/rules/no-named-arguments-with-global-attribute-names.js

```diff
@@ -0,0 +1,179 @@
+'use strict';
+
+const Rule = require('./base');
+const createErrorMessage = require('../helpers/create-error-message');
+const isAngleBracketComponent = require('../helpers/is-angle-bracket-component');
+
+const ERROR_MESSAGE = 'Named argument (%) should not use an HTML global attribute name.';
+
+const DEFAULT_CONFIG = {
+  allow: [],
+  disallow: [],
+};
+
+function makeError(name) {
+  return ERROR_MESSAGE.replace('%', name);
+}
+
+// from the list of global attributes: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes
+// excluding `aria-*`, `data-*`, and event handler attributes like `onclick` (handled separately)
```

Comment:
Can you shed light on why you excluded the `aria-` attributes?

Reply:
Opted for `.startsWith('aria-') ` to check for them on L124 instead of listing them all out, since I was already doing similar for `data-*`

Reply:
Ok, that makes sense. 👍